tensorflow/contrib/nodejs/test/suite.js

```diff
@@ -0,0 +1,7 @@
+var test = require('tape')
+  , tensorflow = require('../index')
```

Comment:
Let's try for the google style guide. As far as I know, nothing resembling
a standard has evolved in the JS community.

On Mon, May 9, 2016 at 10:49 AM Daniel W Mane <notifications@github.com>
wrote:

> In tensorflow/contrib/nodejs/test/suite.js
> <https://github.com/tensorflow/tensorflow/pull/2206#discussion_r62541662>:
>
> > @@ -0,0 +1,7 @@
> > +var test = require('tape')
> > +  , tensorflow = require('../index')
>
> For Python, we are generally following Google style guide, but are using
> the naming conventions from PEP8, to be more consistent with open-source
> users. So Google style guide is a good default, but if there are reasons to
> use a different style guide that's more familiar or widely adopted in the
> nodejs community I think we'd be open to that - especially since I doubt
> think the node bindings will see much use inside of Google.
>
> If you think the google js style guide is a good choice then let's go with
> that.
>
> @martinwicke <https://github.com/martinwicke> feel free to comment more
> authoritatively... (since I know you have such passionate, passionate
> thoughts on javascript styles)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/tensorflow/tensorflow/pull/2206/files/5bb65eeec896ce777a3696982b00fe494e59c846#r62541662>
>